packages/EasyCodingStandard/README.md

```diff
@@ -202,7 +202,7 @@ final class PhptFilesProvider implements ExtraFilesProviderInterface
     {
         # $source is "source" argument passed in CLI
         # inc CLI: "vendor/bin/ecs check /src" => here: ['/src']
-        $finder = Finder::find('*.phpt')->in($source);
+        $finder = Finder::find('*.php', '*.phpt')->in($source);
 
         return iterator_to_array($finder->getIterator());
```

Comment:
This is not related to you commits, but I think about return value of `provideForSource()` method. I remembered a quote from your post: "be open to what you accept, and strict what you provide"

This looks very WTF to me (I know, my idea):

```php
return iterator_to_array($finder->getIterator());
```

Somebody may return:

```php
return $netteFinder; 
// or
return $symfonyFinder;
// or
return $finder->getIterator();
// or string[]
return $files;
```

Maybe we could accept them all and convert to `SplFile[]`. What do you think?